src/OrganisationRegistry/Organisation/OrganisationCommandHandlers.cs

```diff
@@ -933,6 +933,7 @@ private void ThrowIfCircularRelationshipDetected(Organisation organisation, Peri
                         _organisationRegistryConfiguration.Kbo.OrganisationCapacityTypeIdsToTerminateEndOfNextYear,
                         _organisationRegistryConfiguration.Kbo.OrganisationClassificationTypeIdsToTerminateEndOfNextYear,
                         _organisationRegistryConfiguration.Kbo.FormalFrameworkIdsToTerminateEndOfNextYear,
+                        _organisationRegistryConfiguration.VlimpersKeyTypeId,
```

Comment:
This deviates from previous code. let's discuss thistomorrow.

Reply:
we just need to cleanup the config side of things ...